src/test/java/bad/robot/excel/WorkbookResource.java

```diff
@@ -32,6 +34,11 @@ public static Workbook getWorkbook(String file) throws IOException {
         return new PoiWorkbookReader().read(stream);
     }
 
+    public static Workbook getWorkbookFromFactory(String file) throws IOException, InvalidFormatException {
```

Comment:
Not clear why this is new, it seems to do the same as `getWorkbook` unless I'm missing something? I think I prefer the `getWorkbook` as it ensures the stream is closed and (for some reason I've long since forgotten) deals with exceptions in a certain way.

Reply:
Same reason as above.